.editorconfig

```diff
@@ -8,6 +8,9 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[stack.yaml]
+trim_trailing_whitespace = false
```

Comment:
I do not understand the justification behind this change :thinking: ?

Reply:
It breaks the stack2cabal parser and is annoying.